openam-console/src/main/java/com/sun/identity/console/realm/IDRepoViewBean.java

```diff
@@ -56,6 +57,10 @@
 import java.util.Set;
 import javax.servlet.http.HttpServletRequest;
 
+import org.forgerock.http.util.Uris;
+
+import static com.sun.identity.console.XuiRedirectHelper.*;
```

Comment:
Please don't use wildcard imports... https://github.com/WrenSecurity/wrensec-docs/wiki/Java-Style-Guide#331-no-wildcard-imports . Also it reminds me that we might want to enable checkstyle as it is probably not active at the moment and our checkstyle config specifically allows for static wildcard imports :disappointed: .

Reply:
Fixed.